packages/dd-trace/src/appsec/telemetry/telemetry-collector.js

```diff
@@ -0,0 +1,140 @@
+'use strict'
```

Comment:
Should this collector be in the same folder as log collector? `packages/dd-trace/src/appsec/telemetry/api`

Reply:
I've made some changes regarding log-collector